src/components/DisplayBookList.js

```diff
@@ -14,8 +15,8 @@ const DisplayBookList = () => {
 
   return (
     <>
-      <div className="book_cont">
-        <ul>
+      <div className={styles.books_cont}>
+        <ul className={styles.books_min_cont}>
           {books.map((book) => (
             <DisplayAbook
               Id={book.Id}
```

Comment:
[REQUIRED] Kindly ensure that your design matches the template as much as possible.

![image](https://user-images.githubusercontent.com/62938735/214353530-9980d972-a0e3-4111-87a9-0e050a4e800f.png)

Reply:
@fobadara 
Thank you so much for the review.
- [ ] I have ensured that the colours match the design template at this [commit](https://github.com/Lucash2022/Bookstore/commit/727c5a25a6d3c3eae8bac424c2a9deda59055ea5)
- [ ] I have included the image icon at this [commit](https://github.com/Lucash2022/Bookstore/commit/55f36374ea4e120f86b61285f5c4ce5ab4fbe677)

I